translations/ui-orders/fr.json

```diff
@@ -668,4 +668,4 @@
     "checkIn.deletePiece.success": "Piece was deleted successfully",
     "piece.delete.heading": "Delete piece?",
     "piece.delete.message": "Delete piece?"
-}
\ No newline at end of file
+}
```

Comment:
I don't think that these files should be touched at all. We might end in situation when localization script will be changing them back. Please tune up your IDE to ignore those. Thanks